agent/agent.go

```diff
@@ -451,11 +452,12 @@ func (a *Agent) wait(ctx, taskCtx context.Context, tc *taskContext, heartbeat ch
 		if err := tc.oomTracker.Check(oomCtx); err != nil {
 			tc.logger.Execution().Errorf("error checking for OOM killed processes: %s", err)
 		}
-		durationString := fmt.Sprintf("found no OOM kill in %.3f seconds", time.Now().Sub(startTime).Seconds())
-		if found, _ := tc.oomTracker.Report(); found {
-			durationString = fmt.Sprintf("found an OOM kill in %.3f seconds", time.Now().Sub(startTime).Seconds())
+		if lines, _ := tc.oomTracker.Report(); len(lines) > 0 {
+			tc.logger.Execution().Debugf("found an OOM kill (in %.3f seconds)", time.Now().Sub(startTime).Seconds())
```

Comment:
I don't know if dmesg lines correspond 1:1 with processes killed, but would it be possible to include how many OOM kills were found in this log, or does `len(lines)` not correspond to the number of OOM-killed processes?

Reply:
There are potentially more lines that will match for a single OOM kill:
For example, this might be what matches for killing one process
```
[11686.043641] Out of memory: Kill process 2603 (flasherav) score 761 or sacrifice child
[11686.043647] Killed process 2603 (flasherav) total-vm:1498536kB, anon-rss:721784kB, file-rss:4228kB
```
In that case we could still print `len(pids)`, but sometimes it's something like this
```
[11686.040460] flasherav invoked oom-killer: gfp_mask=0x201da, order=0, oom_adj=0, oom_score_adj=0
[11686.040467] flasherav cpuset=/ mems_allowed=0
[11686.040472] Pid: 2859, comm: flasherav Not tainted 3.0.0-12-generic #20-Ubuntu
```
Although we will include the first line, we don't (currently) parse the PID from that message, so `len(pids)` will be 0.